bin/light-base/src/lib.rs

```diff
@@ -1236,3 +1221,112 @@ async fn start_services<TPlat: Platform>(
         transactions_service,
     }
 }
+
+async fn encode_database<TPlat: Platform>(
+    services: &ChainServices<TPlat>,
+    max_size: usize,
+) -> String {
+    // Craft the structure containing all the data that we would like to include.
+    let mut database_draft = SerdeDatabase {
+        chain: match services.sync_service.serialize_chain_information().await {
+            Some(ci) => {
+                let encoded = finalized_serialize::encode_chain(&ci);
+                serde_json::from_str(&encoded).unwrap()
+            }
+            None => {
+                // If the database can't be obtained, we just return a dummy value that will
```

Comment:
are you saying `services.sync_service.serialize_chain_information()`  above obtains the database? maybe you thought to say "If the chain information can't be obtained"?